### DISCUSSION TITLE
I did not receive my perks

### DISCUSSION THREAD

**Discussion by kakarot-dev** [community/community#135471] [GitHub Education]
It's been 4 days since I have been approved for the plan. But it has not sent any updates or allowed me to access my plans. 

I have been told to contact this forum, Please help me.

**Comment by gilsonnogueira** [comment]
I'm in the same situation.

**Comment by davevad93** [comment]
Hi @kakarot-dev ,

The [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits) state that If you haven't received your education benefits within 5 days of your approval status to raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education).

Generally it takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests during busy periods.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kakarot-dev was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
